### DISCUSSION TITLE
Does this have anthropic Prompt Caching

### DISCUSSION THREAD

**Discussion by nomadictuba2005** [cline/cline#223] [Feature Requests]
Exactly that.

**Comment by Charuru** [comment]
Yes

**Comment by SinisterSpatula** [comment]
I added my openrouter api key and it said from the 8 or so models I could pick from that none of them supported prompt caching. Not sure what is up with that. 😢

**Comment by elemental-mind** [comment]
This discussion should be closed

**Comment by Thibaultjaigu** [comment]
We've added prompt caching and initial results show up to 70% cost reduction per task!

Try it out now on the Requesty Router and get $6 free credits! 
https://requesty.ai/router
